rr_endline_detection/src/endline_detection.cpp

```diff
@@ -0,0 +1,108 @@
+#include "endline_detection.hpp"
+
+//constructor
+EndlineCounter::EndlineCounter (ros::NodeHandle nh_) : it_(nh_) 
+{
+  detection_status_ = false;
+  hysteresis_counter_ = 0;
+  hysteresis_constant_ = 2;
```

Comment:
make this into a rosparam so that it can be set from outside

Reply:
need to figure out how